iac/terraform/aws/lambda/variables.tf

```diff
@@ -1,6 +1,6 @@
 variable "environment" {
   type        = string
-  description = "Deployment environment"
+  description = "Deployment environment and resource prefix"
```

Comment:
are you sure its also resource prefix>

Reply:
yes, for `development` branches this will be used as prefix for different resources.